modules/router-store/src/actions.ts

```diff
@@ -14,7 +14,7 @@ import {
 /**
  * An action dispatched when a router navigation request is fired.
  */
-export const ROUTER_REQUEST = 'ROUTER_REQUEST';
+export const ROUTER_REQUEST = '@ngrx/router-store/router_request';
```

Comment:
These can be simplified:
`.../request`,
`.../navigation`,
`.../cancel`,
`.../error`,
`.../navigated`